pages/edit/index.tsx

```diff
@@ -116,6 +116,19 @@ function EditForm({ baseUrl }) {
                 "There was an issue with the " +
                   "cloudflare check. Please try again.",
               );
+            } else if (
+              json.message.includes(`Member with email ${email} not found`)
+            ) {
+              return new Promise((resolve) => {
+                setTimeout(() => {
+                  setPageState(PageState.Error);
+                  setError(
+                    `Looks like the email ${email} is not associated with a ` +
```

Comment:
Small tweak to entertain me: `We don't recognize the email ${email}. Did you sign up with another?`

<img width="558" alt="image" src="https://github.com/user-attachments/assets/a31bb5ce-8850-4003-a379-ee72f8a1a795">

Reply:
updated with commit [457c7a6](https://github.com/hawaiians/hawaiiansintech/pull/36/commits/457c7a64374b991c99047eeee5e38b4e052388fe)